staging/src/k8s.io/kubectl/pkg/cmd/get/BUILD

```diff
@@ -72,11 +73,7 @@ go_test(
         "humanreadable_flags_test.go",
         "sorter_test.go",
     ],
-    data = [
-        "//api/openapi-spec",
-        "//test/e2e/testing-manifests:all-srcs",
-        "//test/fixtures",
-    ],
+    data = ["//staging/src/k8s.io/kubectl/test/data"],
```

Comment:
preexisting, but I'm slightly surprised this wasn't under `test/data`. Any reason not to rename the folder in a follow-up?

Reply:
No. I'll change it in the next PR. This was my fault in a previous PR.